### PR TITLE
Added D2k asset installer d2k-f (German)

### PR DESCRIPTION
**Pull request by Maruhl**
Please review this, it works fine with my original disc.

### CODE REVIEW

File: mods/d2k/mod.yaml [hunk 1]
```diff
@@ -223,7 +223,7 @@ ModContent:
 	Packages:
 		base: Base Game Files
 			TestFiles: ^Content/d2k/v2/BLOXBASE.R8, ^Content/d2k/v2/BLOXBAT.R8, ^Content/d2k/v2/BLOXBGBS.R8, ^Content/d2k/v2/BLOXICE.R8, ^Content/d2k/v2/BLOXTREE.R8, ^Content/d2k/v2/BLOXWAST.R8, ^Content/d2k/v2/SOUND.RS, ^Content/d2k/v2/PALETTE.BIN, ^Content/d2k/v2/FONT.BIN,  ^Content/d2k/v2/FONTCOL.FNT,  ^Content/d2k/v2/FONTCOL.FPL
-			Sources: d2k-a, d2k-a-linux, d2k-b, d2k-b-linux, d2k-c, d2k-c-linux, d2k-d, d2k-d-linux, d2k-e, d2k-e-linux, gruntmods
+			Sources: d2k-a, d2k-a-linux, d2k-b, d2k-b-linux, d2k-c, d2k-c-linux, d2k-d, d2k-d-linux, d2k-e, d2k-e-linux, d2k-f, d2k-f-linux gruntmods
```
Comment (Phrohdoh) [inline mods/d2k/mod.yaml]: Looks like there is a missing comma before `gruntmods` in this list.

**Comment by GraionDilach** [comment]
You need to squash the commits.

**Comment by Maruhl** [comment]
I'm sorry, github and git is completely new to me, I have to learn that everything here works.

File: mods/d2k/mod.yaml [hunk 2]
```diff
@@ -245,5 +245,6 @@ ModContent:
 		d2k|installer/d2k-c.yaml
 		d2k|installer/d2k-d.yaml
 		d2k|installer/d2k-e.yaml
+        d2k|installer/d2k-f.yaml
```
Comment (IceReaper) [inline mods/d2k/mod.yaml]: Spaces instead of tabs?

**Comment by pchote** [comment]
I am very hesitant about this because we have previously established that we will not support any non-english content until #10475 has been resolved.  Otherwise we will be forced to wipe *everybody*'s content in order to support language packs.

**Review by abcdefg30 — COMMENTED** [review]
FWIW, the videos probably won't work with this as-is. `A_BR01_G.VQA` has a `G` for German but we only look for the English version (`A_BR01_E.VQA`) in our mission files.

**Comment by obrakmann** [comment]
Thank you for your PR, but currently it goes against our standing policy of not supporting international versions of the games. Maybe some time in the future.

**Pull request closed without merging**